src/objects.rs

```diff
@@ -1048,6 +1090,110 @@ impl GetObjectType for StopPoint {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub struct StopLocation {
+    pub id: String,
+    pub name: String,
+    #[serde(skip)]
+    pub comment_links: CommentLinksT,
+    pub visible: bool,
+    pub coord: Coord,
+    pub parent_id: Option<String>,
+    pub timezone: Option<String>,
+    pub geometry_id: Option<String>,
+    pub equipment_id: Option<String>,
+    pub level_id: Option<String>,
+    #[serde(skip)]
+    pub stop_type: StopType,
```

Comment:
I'm not a fan of using `StopType` which is already used for `StopPoint`. If I understand well, `StopPoint` can only have the values `StopType::Point` and `StopType::Zone`. And `StopLocation` can only have the values `StopType::StopEntrance`, `StopType::GenericNode` or `StopType::BoardingArea`? If this is the case, why not creating the following `enum`, independent of `StopType`.
```rust
#[derive(Derivative, Debug, PartialEq, Clone)]
pub enum StopLocationType {
    StopEntrance,
    GenericNode,
    BoardingArea,
}
```

Reply:
I agree with @woshilapin 
and may be StopLocation is not a good name